src/utils/iframe.ts

```diff
@@ -30,10 +30,11 @@ export const isIframe = (
 ): element is HTMLIFrameElement =>
   Boolean(element && element.tagName === "IFRAME");
 
+type QueryParamKey = "integrationName";
 export const setIframe = (
   route: string,
   options: Options,
-  params?: Record<string, string>
+  params?: Partial<Record<QueryParamKey, string>>
```

Comment:
Is this the only key left in the record now?

Reply:
Yep! Only using it for integrationName now

Reply:
Yeah we decided to keep this around even though it's the only thing still using query params. It _kinda_ makes sense to leave that as it's specific to the `find-integration` route.